README.md

```diff
@@ -21,15 +21,16 @@ All of them create lots of **great** lines of code, but neither one create a goo
 
 If you're still not convinced, check out these amazing examples:
 
-- [Node Source](https://nodesource.com/404)
-- [Blizzard](http://us.blizzard.com/en-us/not-found.html)
-- [Survey Legend](https://www.surveylegend.com/404)
-- [Lego](http://www.lego.com/404notfound)
-- [CSS Tricks](https://css-tricks.com/thispagedoesntexist)
 - [AirBnb](https://www.airbnb.com.br/pagenotfound)
+- [Blizzard](http://us.blizzard.com/en-us/not-found.html)
 - [Coca Company](http://www.coca-colacompany.com/404/)
 - [CodeSchool](http://codeschool.com/404)
+- [CSS Tricks](https://css-tricks.com/thispagedoesntexist)
+- [Lego](http://www.lego.com/404notfound)
+- [Node Source](https://nodesource.com/404)
 - [Slack](https://slack.com/404)
+- [Spring](https://spring.io/404)
```

Comment:
Thank you for sorting, but please remove _spring.io_ from `README.md` as this is a good example but not so _creative_ to be highlighted in the README!